### PR TITLE
quick add to a couple of pages

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/product/index.njk [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ---
-title: Team-Based Low-Code Development
 layout: layouts/page.njk
+title: Team-Based Low-Code Development
```
Comment (ZJvandeWeg) [inline src/product/index.njk:3]: @Hasmin-AC @MarianRaphael Can we do better here?

Comment (Hasmin-AC) [reply src/product/index.njk:3]: What is this page exactly? I don't see it linked from any where

Comment (ZJvandeWeg) [reply src/product/index.njk:3]: Let's remove it?

Comment (Hasmin-AC) [reply src/product/index.njk:3]: yes I think we should.

**Pull request merged**